actions/release-secrets/action.yml

```
@@ -7,9 +7,12 @@ inputs:
     required: true
   # This uses this format to allow using the GetParameters action.
   ssm_parameter_pairs:
-    description: 'A series of pairs of paths to environment mappings. "/path = ENV_NAME", "/path2 = ENV_NAME2'
+    description: 'A series of pairs of paths to environment mappings. "/path = ENV_NAME, /path2 = ENV_NAME2"'
     required: false
   # If we add S3, then just have a list of S3 paths.
+  s3_path_pairs:
+    description: 'A series of pairs of s3 paths to local paths. The local paths are relative to the project root. "launchdarkly-releaser/dotnet/LaunchDarkly.ClientSdk.snk = LaunchDarkly.ClientSdk.snk"'
```

```suggestion
    description: 'A series of pairs of s3 paths to local paths. The local paths are relative to the project root. "/s3-path = local-path"'
```
(or whatever makes sense as a generic example. To keep it consistent with the example for ssm.)